plugins/modules/web_infrastructure/rundeck_acl_policy.py

```diff
@@ -173,9 +173,9 @@ def create_or_update_acl(self):
             if self.module.check_mode:
                 self.module.exit_json(changed=True, before={}, after=self.module.params["policy"])
 
-            _, info = self.request_rundeck_api("system/acl/%s.aclpolicy" % self.module.params["name"],
-                                               method="POST",
-                                               data={"contents": self.module.params["policy"]})
+            dummy, info = self.request_rundeck_api("system/acl/%s.aclpolicy" % self.module.params["name"],
+                                                   method="POST",
+                                                   data={"contents": self.module.params["policy"]})[1]
```

Comment:
Using both `dummy` and `[1]` won't work.

Reply:
oh darn, missed that one too